project/Dependencies.scala

```diff
@@ -225,14 +225,14 @@ object Dependencies {
     libraryDependencies ++= Seq(
         // https://github.com/googleapis/java-pubsub/tree/master/proto-google-cloud-pubsub-v1/
         "com.google.api.grpc" % "grpc-google-cloud-pubsub-v1" % "1.85.1" % "protobuf-src", // ApacheV2
-        "io.grpc" % "grpc-auth" % "1.28.0", // ApacheV2
+        "io.grpc" % "grpc-auth" % "1.29.0", // ApacheV2
         "com.google.auth" % "google-auth-library-oauth2-http" % "0.20.0", // BSD 3-clause
         // pull in Akka Discovery for our Akka version
         "com.typesafe.akka" %% "akka-discovery" % AkkaVersion
       ) ++ Silencer
   )
 
-  val GooglePubSubGrpcAlpnAgent = "org.mortbay.jetty.alpn" % "jetty-alpn-agent" % "2.0.9"
+  val GooglePubSubGrpcAlpnAgent = "org.mortbay.jetty.alpn" % "jetty-alpn-agent" % "2.0.10"
```

Comment:
Do we still need the ALPN agent?

Reply:
We do for testing on jdk8 before 8u251.

We probably don't do that, so I'll drop it.